src/main/java/fr/eirb/lemondedenemo/periscope/utils/Fish.java

```diff
@@ -0,0 +1,37 @@
+package fr.eirb.lemondedenemo.periscope.utils;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+
+public class Fish {
+  public int length;
+  public int height;
+  public File imageFile;
```

Comment:
Use encapsulation :
```suggestion
  private int length;
  private int height;
  private final File imageFile;
  
  public void setLength(int length) {
    this.length = length;
  }
  
  public int getLength() {
    return this.length;
  }
  
  public void setHeight(int height) {
    this.height = height;
  }
  
  public int getHeight() {
    return this.length;
  }
  
  public File getImageFile() {
    return this.imageFile;
  }
```
If length and height are not modified and won't be modified in the future, please make them final.
This affects other files.